cms/templates/js/add-xblock-component-button.underscore

```diff
@@ -1,8 +1,8 @@
 <% if (type === 'advanced' || templates.length > 1) { %>
-<a href="#" class="multiple-templates add-xblock-component-button" data-type="<%= type %>">
+<button type="button" class="multiple-templates add-xblock-component-button" data-type="<%= type %>">
 <% } else { %>
-<a href="#" class="single-template add-xblock-component-button" data-type="<%= type %>" data-category="<%= templates[0].category %>">
+<button type="button" class="single-template add-xblock-component-button" data-type="<%= type %>" data-category="<%= templates[0].category %>">
 <% } %>
   <span class="large-template-icon large-<%= type %>-icon"></span>
   <span class="name"><%= display_name %></span>
```

Comment:
You may want to add `<span class="sr">Add component: </span>` before the `display_name` to give some actionable context.